packages/ui/src/theme/tokens/components/toggleButton.ts

```diff
@@ -155,14 +157,13 @@ export const togglebutton: ToggleButtonTokens = {
       borderColor: { value: '{colors.border.focus.value}' },
       backgroundColor: { value: '{colors.transparent.value}' },
       boxShadow: { value: '{components.fieldcontrol._focus.boxShadow.value}' },
-      color: { value: '{colors.overlay.50.value}' },
```

Comment:
Was this unused? Isn't it a breaking change to remove this?

Reply:
I think you are right. Even if this doesn't take any effects and inherit from default styling, I should keep it so that it is token customizable.